src/server/status/metrics_collector/metrics.js

```diff
@@ -17,7 +38,7 @@ export class Metrics {
     const metrics = {
       last_updated: timestamp,
       collection_interval_in_millis: this.config.get('ops.interval'),
-      uptime_in_millis: process.uptime() * 1000,
+      uptime_in_millis: process.uptime() * 1000, // TODO: deprecate this field, should be process.uptime_ms
```

Comment:
nit, in theory this may end up with a slightly different number.  thoughts on setting this to process.uptime_ms?

Reply:
yeah, good idea. It makes the TODO make more sense. 0abf0ff387